src/lib/notifications.ts

```
@@ -0,0 +1,60 @@
+
+  if (Platform.OS === 'android') {
+    await Notifications.setNotificationChannelAsync('shapeshiftAndroidChannel', {
+      name: 'A channel is needed for the permissions prompt to appear',
+      importance: Notifications.AndroidImportance.MAX,
+      vibrationPattern: [0, 250, 250, 250],
+      lightColor: '#FF231F7C',
+    });
+  }
+
+  if (Device.isDevice) {
+    const { status: existingStatus } = await Notifications.getPermissionsAsync();
```

ESLint

[error] Delete `;`

(prettier/prettier)